ember_debug/data-debug.js

```diff
@@ -36,34 +39,33 @@ var DataDebug = Ember.Object.extend(PortMixin, {
 
   namespace: null,
 
-  port: Ember.computed.alias('namespace.port'),
-  application: Ember.computed.alias('namespace.application'),
-  objectInspector: Ember.computed.alias('namespace.objectInspector'),
+  port: alias('namespace.port'),
+  application: alias('namespace.application'),
+  objectInspector: alias('namespace.objectInspector'),
 
   portNamespace: 'data',
 
-  modelTypesAdded: function(types) {
-    var self = this, typesToSend;
-    typesToSend = types.map(function(type) {
-      return self.wrapType(type);
+  modelTypesAdded(types) {
+    let typesToSend;
+    typesToSend = types.map((type) => {
+      return this.wrapType(type);
     });
```

Comment:
Tip: single-line arrow functions can have implicit returns if you don't use curly brackets:
```js
types.map(type => this.wrapType(type));
```